config/worker.rb

```diff
@@ -21,7 +21,10 @@
 ActiveJob::Base.queue_name_delimiter = '_'
 
 # load the code
-['lib', 'app/processors', 'app/workers'].each do |path|
+[ 'lib',
+  'app/processors/concerns', 'app/processors',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.